### PR TITLE
Feat/evidence attachment display

### PR DESCRIPTION
**Pull request by Harman-singh-waraich**
## PR-Codex overview
This PR introduces new features and improvements related to file viewing and attachment display in the application.

### Detailed summary
- Increased `z-index` in the `Header` component
- Added a hidden class for `@cyntler/react-doc-viewer`
- Updated sorting criteria in `useEvidences` query
- Added new routes and components for evidence attachment display
- Styled custom scrollbars
- Added new SVG icons for new tab and arrow left
- Implemented Markdown file rendering functionality
- Created a loader component
- Styled the header for attachment display
- Updated `EvidenceCard` component with new functionalities

> The following files were skipped due to too many changes: `web/src/components/EvidenceCard.tsx`, `yarn.lock`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by kemuru — COMMENTED** [review]
1) I want to be able to open the source file in a new tab (not download it), its more comfortable for me, how would I do that
2) would it be possible to add a dark mode? maybe inverting colors depending on the {theme} the user is in (light or dark mode)
image

**Comment by Harman-singh-waraich** [comment]
> 1. I want to be able to open the source file in a new tab (not download it), its more comfortable for me, how would I do that
> 2. would it be possible to add a dark mode? maybe inverting colors depending on the {theme} the user is in (light or dark mode)
> 


1. We can add a "Open in new tab" link or just the icon .
2. I have already configured dark theme for the markdown and text. As for Images, i kept the white background so it's easier to see the boundaries. For PDF, we cannot set dark theme

**Review by kemuru — DISMISSED** [review]
lgtm now 🤔 can replicate in escrow-v2 later

File: web/src/components/FileViewer/Viewers/MarkdownViewer.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    color: ${({ theme }) => theme.secondaryText};
+  }
+`;
+
+const MarkdownRenderer: DocRenderer = ({ mainState: { currentDocument } }) => {
+  if (!currentDocument) return null;
+  const base64String = (currentDocument.fileData as string).split(",")[1];
+
+  // Decode the base64 string
+  const decodedData = atob(base64String);
```
Comment (alcercu) [inline web/src/components/FileViewer/Viewers/MarkdownViewer.tsx:26]: maybe let's `useMemo` here just in case we run into big files 🤔

**Pull request merged**